packages/emulators/libretro/puae2021-lr/package.mk

```diff
@@ -0,0 +1,46 @@
+################################################################################
+#      This file is part of OpenELEC - http://www.openelec.tv
+#      Copyright (C) 2009-2012 Stephan Raue (stephan@openelec.tv)
+#      Copyright (C) 2020      351ELEC team (https://github.com/fewtarius/351ELEC)
+#      Copyright (C) 2023-present Fewtarius
```

Comment:
You should add your own copyright here since you worked on the package. 😁

Reply:
I changed the Copyright https://github.com/JustEnoughLinuxOS/distribution/commit/828f1b0430968f9e7f2036cb9c1413483830e8e4 however I'm not familiar with the Github review process. Is the update I just did enough or should I make a new pull request?